src/wp-includes/media.php

```diff
@@ -1056,6 +1057,11 @@ function wp_get_attachment_image( $attachment_id, $size = 'thumbnail', $icon = f
 
 		$attr = wp_parse_args( $attr, $default_attr );
 
+		// Omit the `decoding` attribute if empty or false.
+		if ( array_key_exists( 'decoding', $attr ) && empty( $attr['decoding'] ) ) {
```

Comment:
This should prevent invalid values such as `decoding="invalid"`.

```suggestion
		if ( array_key_exists( 'decoding', $attr ) && ! in_array( $attr['decoding'], array( 'async', 'sync', 'auto' ), true ) ) {
```

Reply:
👍🏼 we do something similar where we support decoding